File: Iniciante/3302.cpp
#include <iostream>
using namespace std;

int main() {
    int n, p;
    cin >> n;
    for(int i = 1; i <= n; i++) {
        cin >> p;
        cout << "resposta " << i << ": " << p << endl;
    }
    return 0;
}

File: Iniciante/C#/3302.cs
using System;
class URI {
    static void Main(string[] args) { 
        int n = int.Parse(Console.ReadLine());
        for(int i = 1; i <= n; i++) {
            int p = int.Parse(Console.ReadLine());
            Console.WriteLine("resposta " + i + ": " + p);
        }
    }   
}